docs/slotfilling.rst

```diff
@@ -192,3 +192,254 @@ like ``MyCustomFormField`` and overriding the ``validate()`` method.
    Typically your NLU model will assign this free-text input to 2-3 different intents. 
    It's easiest to add stories for each of these. 
 
+Slot Filling with Forms
+-----------------------
+
+Another alternative is to use Forms, an object which supercedes the policy ensemble and carries out deterministic logic for the purpose of filling slots.
```

Comment:
"supercedes the policy ensemble and carries out deterministic logic for the purpose of filling slots" - this is super complex english and not v readable for non-native speakers. Let's stick to plain english